metadata-ingestion/src/datahub/ingestion/source_report/usage/bigquery_usage.py

```diff
@@ -0,0 +1,13 @@
+import collections
+import dataclasses
+from typing import Counter
+
+from datahub.ingestion.api.source import SourceReport
+
+
+@dataclasses.dataclass
+class BigQueryUsageSourceReport(SourceReport):
```

Comment:
1. Nice to see this getting pulled out into its own file! Name it bigquery_usage_source_report.py?
2. Add more details from config similar to the big query report?

Reply:
This file is in the `source_report` folder. Seems redundant to rename it. 
Will add more details from config.

Reply:
added more configs to report